.github/workflows/agw-build-publish-container.yml

```diff
@@ -188,3 +188,69 @@ jobs:
+    if: always() && github.event_name == 'push'
+    needs: [test-containers-precommit, test-containers-extended]
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # pin@v3.1.0
+
+      - name: Collect all test results
+        run: |
+          mkdir -p lte/gateway/test_results
+          if [ -n ${{ needs.test-containers-precommit.outputs.final_status }} ]
+            cp ${{ needs.test-containers-precommit.outputs.test_results }} lte/gateway/test_results/
+            echo ${{ needs.test-containers-precommit.outputs.final_status }} > test_status_precommit.txt
+          else
+            echo fail > test_status_precommit.txt
+          fi
+          if [ -n ${{ needs.test-containers-extended.outputs.final_status }} ]
+            cp ${{ needs.test-containers-extended.outputs.test_results }} test_results/
+            echo ${{ needs.test-containers-extended.outputs.final_status }} > test_status_extended.txt
+          else
+            echo fail > test_status_extended.txt
+          fi
+
+      - name: Determine end result for both tests
+        run: |
+          diff -q test_status_precommit.txt test_status_extended.txt > /dev/null
+          if [ $? -eq 0 ]
+          then
+            mv test_status_precommit.txt test_status.txt
+          else
+            echo fail > test_status.txt
+          fi
+          rm test_status_*.txt
+
+      - name: Setup python
+        uses: actions/setup-python@13ae5bb136fac2878aff31522b9efb785519f984 # pin@v4.3.0
+        with:
+          python-version: '3.8.10'
+
+      - name: Install pre requisites
+        run: |
+          pip3 install --upgrade pip
+          pip3 install firebase_admin
+
+      - name: Publish results to Firebase
+        env:
+          FIREBASE_SERVICE_CONFIG: ${{ secrets.FIREBASE_SERVICE_CONFIG }}
+          REPORT_FILENAME: "lte_integ_test_containerized${{ github.sha }}.html"
+        run: |
+          npm install -g xunit-viewer
+          [ -d "lte/gateway/test-results/" ] && { xunit-viewer -r lte/gateway/test-results/ -o "$REPORT_FILENAME"; }
+          [ -f "$REPORT_FILENAME" ] && { python ci-scripts/firebase_upload_file.py -f "$REPORT_FILENAME" -o out_url.txt; }
+          [ -f "out_url.txt" ] && { URL=$(cat out_url.txt); }
+          python ci-scripts/firebase_publish_report.py -id ${{ github.sha }} --verdict ${{ job.status }} --run_id ${{ github.run_id }} containerized_lte --url $URL
+
+      - name: Notify failure to slack
+        if: failure()
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+          SLACK_USERNAME: ${{ github.workflow }}
+          SLACK_AVATAR: ":boom:"
+        uses: Ilshidur/action-slack@689ad44a9c9092315abd286d0e3a9a74d31ab78a # pin@2.1.0
+        with:
+          args: "Containerized LTE integration test test failed on [${{ github.sha }}](${{github.event.repository.owner.html_url}}/magma/commit/${{ github.sha }}): ${{ steps.commit.outputs.title}}"
```

Comment:
Typo: `test test`

Reply:
Copy-paste-bug. I'll open a PR to fix it in the other tests as well :+1: